LoadingUISDK/LoadingUISDK/ViewController.swift

```diff
@@ -7,12 +7,15 @@
 //
 
 import UIKit
+import LoadingUIFramework
 
 class ViewController: UIViewController {
 
+    @IBOutlet weak var loadingUIButton: UIButton!
```

Comment:
This does not need to be weak
```suggestion
    @IBOutlet var loadingUIButton: UIButton!
```